app/webpack/javascripts/modules/Modules.AllocationDataTable.js

```diff
@@ -126,7 +126,7 @@ moj.Modules.AllocationDataTable = {
                 '<div class="govuk-checkboxes__item">' +
                 '<input class="govuk-checkboxes__input dt-checkboxes" type="checkbox" value="" name="claim-' + data + '" id="claim-' + data + '">' +
                 '<label class="govuk-label govuk-checkboxes__label" for="claim-' + data + '">' +
-                '<span class="govuk-visually-hidden">Select claim ' + data + '</span>' +
+                '<span class="govuk-visually-hidden">Select case ' + row.case_number + '</span>' +
```

Comment:
should this not match the language "claim" that is used else-where

Reply:
`row.case_number` return a case number, for example T20170101, therefore I thought it would be more descriptive for assistive technology users if it read 'Select case T20170101' instead of 'Select claim T20170101'.

Also note, the next table column describes this number format as 'case', not 'claim'.

As for it's use elsewhere, I will make these changes as they arise.


@jsugarman